docs/examples/steps.yaml

```diff
@@ -0,0 +1,29 @@
+---
+Steps:
+- Name: "emr-setup"
+  HadoopJarStep:
+    Args:
+    - "s3://$S3_CONFIG_BUCKET/component/emr-setup.sh"
+    Jar: "s3://eu-west-2.elasticmapreduce/libs/script-runner/script-runner.jar"
+  ActionOnFailure: "CONTINUE"
+- Name: "copy-hbase-configuration"
+  HadoopJarStep:
+    Args:
+    - "bash"
+    - "-c"
+    - "sudo cp /etc/hbase/conf/hbase-site.xml /etc/spark/conf/"
+    Jar: "command-runner.jar"
+  ActionOnFailure: "CONTINUE"
+- Name: "submit-job"
+  HadoopJarStep:
+    Args:
+    - "spark-submit"
+    - "s3://$S3_CONFIG_BUCKET/component/analytical-dataset-generation/generate-analytical-dataset.py"
+    - "--deploy-mode"
+    - "cluster"
+    - "--master"
+    - "yarn"
+    - "--conf"
+    - "spark.yarn.submit.waitAppCompletion=true"
+    Jar: "command-runner.jar"
+  ActionOnFailure: "CONTINUE"
```

Comment:
Missing EOL character at the end of this file. Your editor should have a setting to force this on save.

Reply:
Fixed by https://github.com/dwp/emr-launcher/pull/2/commits/91a7da25fba7119fb83d04f577b6956d6818d824